apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -171,7 +184,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
                 else:
                     html_params["incorrect"] = True
             except Exception as exc:
-                raise ValueError("invalid score" + score) from exc
+                raise ValueError("invalid score" + str(score)) from exc
```

Comment:
This would render something like `invalid scorefoobar`. This is repeated across several other elements. This should probably be something like:

```suggestion
                raise ValueError(f"invalid score: {score}")) from exc
```

Reply:
Changed in de3f21099d117a444d8f8d12bb2d609a63be0443.